### DISCUSSION TITLE
Actionbar after enter a claimed/wilderness chunk not showing up after switch to 1.20.4

### DISCUSSION THREAD

**Discussion by EpicuriousDev** [TownyAdvanced/Towny#7126] [Q&A] [answered]
Hey everyone, the actionbar that normally shows up after you enter a claimed chunk or wilderness chunk, is not showing up anymore after we updated to 1.20.4. The update seems to have messed with a couple of our config files, do you guys know a fix?

**Comment by LlmDl** [comment, marked as answer]
We likely just have to compile Towny with the updated Adventure.

**Reply by EpicuriousDev** [reply]
So as server owners we can not fix it right now?

**Reply by LlmDl** [reply]
I will probably get a pre-release out today, hopefully that fixes it for you.

**Reply by EpicuriousDev** [reply]
Thanks!
